examples/async-universal/common/routes.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import Route from 'react-router/lib/Route'
+
+import Reddit from './containers/Reddit'
+
+export default(
+  <Route path="/" component={Reddit}>
+    <Route path=":id" component={Reddit}/>
```

Comment:
This config is a bit weird. Routes describe component nesting structure. `Reddit` being *inside* `Reddit` doesn’t quite make sense to me. I’m not very good at router configs these days, but I think there should be a way to express this more correctly.